js_modules/dagster-ui/packages/ui-core/src/assets/useWipeAssets.tsx

```diff
@@ -66,13 +105,43 @@ export function useWipeAssets({
     setIsWiping(false);
   };
 
+  const backgroundWipeAssets = async (assetPartitionRanges: PartitionsByAssetSelector[]) => {
+    if (!assetPartitionRanges.length) {
+      return;
+    }
+    setIsWiping(true);
+    const result = await requestBackgroundAssetWipe({
+      variables: {assetPartitionRanges},
+      refetchQueries,
+    });
+    const data = result.data?.backgroundWipeAssets;
+    switch (data?.__typename) {
+      case 'AssetNotFoundError':
+      case 'PythonError':
+        setFailedCount(assetPartitionRanges.length);
+        onComplete?.();
+        setIsWiping(false);
+        return;
```

Comment:
It would be helpful to surface the errors.

```
case 'AssetNotFoundError': {
  showCustomAlert({
    title: 'Asset not found',
    description: 'The specificed asset could not be found.',
  });
  setFailedCount(...);
  // etc.
  return;
}
case 'PythonError': {
  showCustomAlert({
    title: 'Python error',
    body: <PythonErrorInfo error={runs} />,
  });
  // etc.
}
```